responsive-web-design/build-survey-form/index.html

```diff
@@ -0,0 +1,195 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Survey Form</title>
+    <link rel="stylesheet" type="text/css" href="desing.css">
+</head>
+
+<body>
+
+    <h1 id="title">Survey Form</h1>
+
+    <div id='center'>
```

Comment:
This name looks very general, can you change it and add a name more descriptive, like `container-survey-form`

Reply:
You're right. 

Clean Code. Chapter 2. Meaningful names.